source/code-generation/query-converter.txt

```diff
@@ -0,0 +1,23 @@
+.. _rm-query-converter:
+
+===============
+Query Converter
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Query Converter allows you to convert your SQL statements, views, and 
```

Comment:
[minimalism]

I might consider either:

- Query Converter lets you convert SQL statements...
- Use Query Converter to convert SQL statements...